### ISSUE TITLE
Add Linux ARM Binaries for Download

### ISSUE THREAD

**Issue by xiegeo** [visualfc/liteide#680]
ARM is not currently listed under supported systems, but it builds and runs successfully ( I tested on Raspberry Pi 3, ARMv7, Raspbian(Debian Jessie) , Qt4)

**Comment by visualfc** [comment]
Thanks, but I did not any arm compiler environment

**Comment by xiegeo** [comment]
If you want, I can give you a rpi3 (not sure of the logistics). 

Or add ARM as supported in the docs, but source only, maybe with links to binaries released by other people.

**Comment by petergloor** [comment]
Over the past 9 months I've built it many times on Raspbian Wheezy and Raspbian Jessie Lite on a Raspberry Pi 2 Model B. I dont remember, but I think I started with X27 or X28. It works even better and faster with QT5. I'm using it with PuTTY and VcXsrv from my Windows PC. Currently there is an error message at startup, but it works without any problems. I've seen the same message before. The message was not there in all my builds, but it might also be a configuration issue.

Building from a µSD card can be a nightmare and slow. Therefore I'm booting into an attached USB Drive and I'm building from there. I'm not sure as I dont measure the time, but I think this takes less than an hour and will certainly be faster on a RPi 3. Simply make sure to update Raspbian Jessie and follow the instructions for Ubuntu, it works like a charm for me.

**Comment by petergloor** [comment]
I experienced the issue mentioned in my previous comment with any version of QT5 I tested on Linux (up to 5.3.2 32-bit ARM on Raspbian Jessie and 5.5.1 64-bit on Ubuntu 16.04.1 LTS) over Windows X servers on Windows 10 using PuTTY and VcXsrv. 

Meanwhile I found the issue also reported at https://bugreports.qt.io/browse/QTBUG-48453 and after some testing I can assert that the problem appears in other applications as well and is not a specific bug in LiteIDE. 

The problem disappeared after I reinstalled PuTTY to the currently latest release 0.67 and VcXsrv to the currently actual release 1.18.3.0 downloaded from its original source at https://sourceforge.net/projects/vcxsrv/.

Before the update I completely uninstalled the previous versions including saved sessions. As such I also had a clean install and configuration.

**Comment by krolaw** [comment]
Being able to code Go on a PineBook (https://www.pine64.org/?page_id=3707) would be awesome.  (I only use LiteIDE).  Cheap enough to not worry too much about when on holiday.  Is someone able to host their ARM builds?

**Comment by zengming00** [comment]
可以使用qemu在PC上模拟，或者使用linuxdeploy在安卓手机上安装linux环境VNC连接进去，前几天刚刚尝试了这种方法，能编译成功，但是运行后界面显示有很多问题

**Comment by zhuwei** [comment]
我也希望能加个arm的二进制执行文件。我在rpi3上编译不通过。。。

**Comment by xiegeo** [comment]
@zhuwei  我试了试最新的版本，没问题。你按这个做https://github.com/visualfc/liteide/blob/master/liteidex/deploy/welcome/en/install.md#raspbian-jessie 看看在那一步卡住了。

这个是我编译出来的https://1drv.ms/u/s!AoYgncc810e2gdBkpPJfZitxcnZogA

**Comment by zhuwei** [comment]
@xiegeo 谢谢，我试试

**Comment by dtrehas** [comment]
xiegeo, do you have any newer build? (e.g. LiteIDE 36.1 )

**Comment by xiegeo** [comment]
No, but I can help you if you run into any trouble building it.

**Comment by cui-wei** [comment]
同求 linux arm 版本。

**Comment by cui-wei** [comment]
> 可以使用qemu在PC上模拟，或者使用linuxdeploy在安卓手机上安装linux环境VNC连接进去，前几天刚刚尝试了这种方法，能编译成功，但是运行后界面显示有很多问题

我也试过在arm的笔记本上编译，能运行，很慢，界面布局也有点乱。
